wrangler/src/@utils/do-tx-fees.js

```diff
@@ -19,7 +29,47 @@ export default class TxFees {
       this.value = typeof this.value !== 'object' ? iv : this.value
     }
 
-    if (method === 'POST') {
+    if (segment_1) {
+
+      if (method === 'POST') {
+        const transactionHash = segment_1
```

Comment:
this naming threw me off for a bit.

We should probably call it authHash? Since this function is being called with `POST /${authHash}`

```
await txFeesStub.fetch(`/${authedHash}`, {method: 'POST'}).then(handleResponse)
```

Reply:
Well in one instance `segment_1` is a public key, in this instance it's a transaction hash. I like to name things based on file context and while I guess you could go either way here it makes more sense to know what type of data is coming in vs at other times. In this case it's a transaction hash not a stellar public key. In the run.js file it's an authhash in context so that name makes sense. idk. I could see either approach.